### DISCUSSION TITLE
Validation error for dynamodb stream trigger with stream ARN from lift plugin

### DISCUSSION THREAD

**Discussion by jcranendonk** [serverless/serverless#11134] [Q&A]
I'm running into the issue reported here: https://github.com/getlift/lift/issues/209 

Is this an issue with the lift plugin, or in the serverless configuration validation schema?

**Comment by mnapoli** [comment]
Hi @jcranendonk, I think it's worth continuing the discussion in the Lift issue for now.

**Comment by jcranendonk** [comment]
@mnapoli Is it actually a Lift issue, though? A [comment](https://github.com/getlift/lift/issues/209#issuecomment-1136494101) on that thread suggests it may have something to do with the order of variable resolution and service definition validation in Serverless.
